src/resources/coupons-resource.js

```diff
@@ -1,53 +1,52 @@
-export default function CouponsResource({apiHandler}) {
-    return {
-        getAll({limit = null, offset = null, sort = null, filter = null, q = null} = {}) {
-            const params = {
-                limit,
-                offset,
-                sort,
-                filter,
-                q
-            };
-            return apiHandler.getAll(`coupons`, params);
-        },
-
-        get({id}) {
-            return apiHandler.get(`coupons/${id}`);
-        },
-
-        create({id = '', data}) {
-            return apiHandler.create(`coupons/${id}`, id, data);
-        },
-
-        update({id, data}) {
-            return apiHandler.put(`coupons/${id}`, data);
-        },
-
-        getAllRedemptions({limit = null, offset = null, sort = null, filter = null, q = null} = {}) {
-            const params = {
-                limit,
-                offset,
-                sort,
-                filter,
-                q
-            };
-            return apiHandler.getAll(`coupons-redemptions`, params);
-        },
+/**
+*This file was auto-generated by our auto-sdk-code-generator.
```

Comment:
```suggestion
* This file was auto-generated by our auto-sdk-code-generator.
```

Just to be super pedant :)